setup.cfg

```diff
@@ -13,7 +13,7 @@ install_requires =
   fastapi==0.86.0
   httpx==0.22.0
   protobuf==3.19.4
-  rohmu>=2.1.0,<2.2
+  rohmu>=2.2.0,<2.3
```

Comment:
The upper bound should be `<3`. All 2.X versions are supposed to be backward compatible and it's annoying to update all projects using rohmu every time we make a new compatible rohmu release (like we had to do for robjecter).